cmd/agent/app/dogstatsd_stats.go

```diff
@@ -95,8 +97,16 @@ func requestDogstatsdStats() error {
 	}
 
 	if dsdStatsFilePath != "" {
+		// if the file is already existing, ask for a confirmation.
+		if _, err := os.Stat(dsdStatsFilePath); err == nil {
+			if !input.AskForConfirmation(fmt.Sprintf("'%s' existing, do you wan't to overwrite it?", dsdStatsFilePath)) {
```

Comment:
Two nitpicks here:
- For consistency, we could also append `[Y/N]` after the question mark
- It would be a good time to remove nested conditionals by asserting the opposite statement and returning early. WDYT?
```go
// Write to stdout
if dsdStatsFilePath == "" {
    fmt.Println(s)
    return nil
}

// Write to file
if _, err := os.Stat(dsdStatsFilePath); err == nil {
...
```

Reply:
100% agree with both points, I've addressed them.